components/credentials/credentials-controller.js

```diff
@@ -20,6 +33,58 @@ function factory($scope, config, $location, ipCookie, localStorageService) {
   var trustedDomainPrefix = 'settings.trustedDomains.';
   var trustedDomainKey = trustedDomainPrefix + self.callbackHostName;
 
+  if(
+    self.identity.id !== self.requestOwner &&
```

Comment:
We treat `if` differently from functions ... we don't drop all conditions down a line like we do for all parameters, keep first condition on the same line as `if(`.